emulated_hue/__main__.py

```diff
@@ -70,6 +70,8 @@
     token = args.token
     if args.verbose or os.getenv("VERBOSE", "").strip() == "true":
         logger.setLevel(logging.DEBUG)
+    # turn down logging for hass-client
+    logging.getLogger("hass_client").setLevel(logging.INFO)
```

Comment:
Can we make this respect the verbose logging flag?

Reply:
Well, rather not, because it is REALLY verbose :-)
debug logging on the hass client is more interesting when actually debugging the hass client